lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/profiler/WakeProfiler.java

```diff
@@ -44,6 +46,12 @@
   private final Map<InjectionFuture<?>, Object> futures = new MonotonicHashMap<>();
   private final Map<Object, Stats> stats = new MonotonicHashMap<>();
 
+  /**
+   * Parameter to enable Wake network profiling. By default profiling is disabled.
+   */
+  @NamedParameter(doc = "If true, profiling will be enabled", short_name = "profiling", default_value = "false")
+  public static final class ProfilingEnabled implements Name<Boolean> { }
```

Comment:
Please move this to a class of its own.

*Reason:* Tang is interoperable between Java and C#, and the naming schemes of inner classes are not compatible between the two worlds.